styles.css

```diff
@@ -73,10 +87,204 @@ main {
 li {
   list-style-type: none;
   color: #505f79;
-  transition: 0.3s all ease-in-out;
 }
 
 li :hover {
-  transform: rotate(30deg);
   color: #6070ff;
 }
+
+/* Work-Station Section */
+.work-section {
+  display: grid;
+  grid-template-columns: minmax(327px, 100%);
+  grid-template-rows: repeat(3, 1fr);
+  grid-auto-rows: 718px;
+  grid-row-gap: 88px;
+  margin: 0 24px 88px;
+}
+
+.padding {
+  padding: 10px;
+  background-color: #fff;
+  border-radius: 16px;
+}
+
+.frontpage-icon {
+  width: 295px;
+  height: 220px;
+  margin: 0 auto 0 auto;
+}
+
+h2 {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.tonic-section-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  width: 236px;
+  height: 32px;
+  gap: 12px;
+  padding: 0;
+}
+
+.canopy {
+  color: #344563;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.counter {
+  color: #c1c7d0;
+  width: 8px;
+  height: 8px;
+}
+
+.back-end-dev {
+  color: #7a869a;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
```

Comment:
[OPTIONAL] 
- [ ] Kindly make this inline
![image](https://user-images.githubusercontent.com/29071984/170832363-1e0616fd-aafd-46b2-8f1f-dd06d89ee472.png)